docs/src/modules/java/pages/actions-as-controller.adoc

```diff
@@ -20,21 +24,46 @@ include::example$java-valueentity-counter/src/main/proto/com/example/actions/dou
 <2> The protobuf option (akkaserverless.service) is specific to code-generation as provided by the Akka Serverless Maven plugin. This annotation indicates to the code-generation that an Action must be generated.
 <3> The Action mimics the Counter API
 
+[.tabset]
```

Comment:
this `[.tabset]` should be removed, it should only be one at the top, and then `Java::` ... `Scala::`

Reply:
Thanks